### PR TITLE
Updated speech2srt

### PR DESCRIPTION
**Pull request by eric-foy**
Used the 2.0.0 migration guide https://github.com/googleapis/python-speech/blob/master/UPGRADING.md#enums-and-types

nanos not found in datetime.timedelta, used microseconds instead.
Converted to milliseconds.

### CODE REVIEW

**Comment by eric-foy** [comment]
@googlebot I signed it!

________________________________________

**Comment by ToddKopriva** [comment]
Thanks for the update, @eric-foy .

@lepistom , can you have a look at this suggestion regarding code that you contributed?

**Comment by ToddKopriva** [comment]
The changes look good to me, but I'll hold off on approving until @lepistom has approved.

**Comment by lepistom** [comment]
LGTM!

**Review by lepistom — APPROVED** [review]
LGTM

**Review by ToddKopriva — APPROVED** [review]

**Pull request merged**
